ch03_Businessarchitectuur.md

```diff
@@ -156,6 +156,7 @@ Het gebruik van een Organisatie Identificatienummer (OIN) beoogt rechtspersonen
 ### Handelsregister
 
 Een OIN is -waar mogelijk- afgeleid van bestaande identificerende nummers uit het Handelsregister.
+Indien een organisatie een FI nummer én een nummer in het handelsregister heeft (een KvK nummer) dan is het nummer uit het handelsregister leidend.
```

Comment:
Digikoppeling Identificatie en Authenticatie 1.4.2 Geeft aan dat RSIN leidend is voor overheidsorganisaties
https://publicatie.centrumvoorstandaarden.nl/dk/idauth/#identiteit-en-nummer

Reply:
Letterlijke tekst
_De inhoud van het HR is door de Wet op het Handelsregister uitgebreid met rechtspersonen met een publieke taak. Het HR onderkent in de structuur drie niveaus:_
- _Het niveau rechtspersoon met als identificatienummer het Rechtspersonen en Samenwerkingsverbanden Identificatie Nummer (RSIN). Dit is inhoudelijk gelijk aan het Fiscaal (FI)-nummer. Het HR heeft aangegeven, dat dat niveau, en dus dat nummer, in principe het meest geschikt is om voor identificatie op Digikoppeling te gebruiken._
- _Het niveau onderneming/maatschappelijke activiteit met als identificatienummer het KvKnummer. Aangegeven is, dat dat nummer naar alle waarschijnlijkheid voor de overheid als OIN (dus op Digikoppeling) niet geschikt is. Voor bedrijven en instellingen is het KvK nummer wel geschikt._
- _Het niveau vestiging, een relatief zelfstandig onderdeel binnen rechtspersoon en/of onderneming. Het vestigingsbegrip is Europees geregeld en bleek daarom niet bruikbaar voor gebruik binnen de OIN Systematiek._
Het OIN Stelsel Document moet hier dus op aangepast worden.